forge-gui/res/adventure/Innistrad/printsheets.txt

```diff
@@ -0,0 +1,763 @@
+
+#----------------------SHEN CUSTOM JUMPSTART PACKS START HERE---------------------------------#
+
+[ISD Dearly Departed 1]
```

Comment:
Is this duplicated? Maybe we should try to load printsheets from its default location as well as res/adventure/*/printsheets.txt? As well?